pkgs/applications/version-management/git-and-tools/gita/default.nix

```diff
@@ -1,24 +1,38 @@
 { lib
 , buildPythonApplication
 , fetchPypi
+, fetchurl
 , pyyaml
 , setuptools
+, installShellFiles
 }:
 
 buildPythonApplication rec {
-  version = "0.10.5";
+  version = "0.10.9";
   pname = "gita";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1xggslmrrfszpl190klkc97fnl88gml1bnkmkzp6aimdch66g4jg";
+    sha256 = "0fbzk9rj895s5fpbnsyy3gxwbf5spqycisx5cqwzxgm0n5qkz9dk";
   };
 
   propagatedBuildInputs = [
     pyyaml
     setuptools
   ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion --bash --name gita ${
+      fetchurl {
```

Comment:
looks like the completions are part of the repository, but aren't in the pypi tarball. 
Would you mind replacing `fetchPypi` with `fetchFromGitHub`? and install the completions from there

Reply:
Done.  Also I enabled tests if you don't mind.